docs/dev_guide/diagnostic_ref/dpct1074.rst

```diff
@@ -49,6 +57,6 @@ which is manually adjusted to:
 .. code-block:: cpp
    :linenos:
   
-     tex.set(sycl::coordinate_normalization_mode::normalized);
+     tex->set(sycl::coordinate_normalization_mode::normalized);

```

Comment:
can it be more specific on how to write the normalization?

Reply:
done